pybamm/batch_study.py

```diff
@@ -196,7 +197,8 @@ def create_gif(self, number_of_images=80, duration=0.1, output_filename="plot.gi
             Name of the generated GIF file.
 
         """
-
+        if not self.sims:
```

Comment:
You could do something like -
```suggestion
        if not hasattr(self, "sims"):
```
I think this should work.